core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeBean.java

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.employee;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+public class EmployeeBean implements Serializable {
```

Comment:
just `Employee`? Why Bean?
```suggestion
public class Employee implements Serializable {
```

Reply:
> 

We are showing the conversion of POJO to a JavaBean. Using the name EmployeeBean gives more context to the readers.